charts/honeycomb/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Honeycomb Kubernetes Agent Helm Chart Changelog
 
+## Honeycomb Kubernetes Agent v1.8.0
+ - maint: bump honeycomb chart appVersion (#242) | @fchikwekwe
```

Comment:
I believe the only change since chart version 1.7.1 was https://github.com/honeycombio/helm-charts/pull/242. Please update the changelog to highlight that change.

Reply:
Also, please remove the entry about bumping the appVersion since we didn't do that.

Reply:
@TylerHelmuth what do you mean? Do you want me to just state that #242 was the last change?

Reply:
Ya.  Including `- maint: bump honeycomb chart appVersion (#242) | @fchikwekwe` is incorrect because the appVersion was not bumped.  We should remove that line.

Reply:
We don't include the prep release PR in the changelog.

Reply:
Okay, I understand. Thank you.